example.cfg

```diff
@@ -84,6 +84,7 @@ SVN_DEFAULT_IGNORES = """
 COMMIT_MESSAGE_FILTERS = [
 	agito.reflow_text,         # Reflow text to 72 columns
 	agito.append_branch_info,  # Append SVN branch/rev info.
+	agito.rewrite_commit_refs, # Rewrite rDDD or [DDD] SVN commit refs to Git SHA1 hashes
```

Comment:
it's not Trac specific, it's rather general convention, at least the `r123` syntax